docs/_snippets/recipes_flutter_navigation/navigation2/main.dart.md

```diff
@@ -0,0 +1,99 @@
+```dart
+import 'package:flow_builder/flow_builder.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'bloc/book_bloc.dart';
+
+void main() {
+  runApp(
+    BlocProvider(
+      create: (context) => BookBloc(),
```

Comment:
```suggestion
      create: (_) => BookBloc(),
```

Reply:
Can we provide the `BookBloc` as part of the `BooksApp` rather than providing it as part of `runApp`?